src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -299,6 +299,22 @@
                 https://<deploy_user_name>@MyUniqueApp.scm.azurewebsites.net/MyUniqueApp.git
 """
 
+helps['webapp deployment source config-zip'] = """
```

Comment:
Suggest apply the help on functionapp as well, maybe adding `helps['functionapp deployment source config-zip']=helps['webapp deployment source config-zip']` below?